### PR TITLE
Show Readme file from github in Component.vue page

### LINKED ISSUE #56: load readme into component view page
right now I have to click through to npm to learn the things I want to know - it might be nicer to embed it if possible.

### PR DESCRIPTION
**Pull request by Nikhil-Vats**
Shows readme.md of the component's repository from github on the component page.
fixes #56 
![Image- 1: Zoomed out view, hence looking very small] (https://user-images.githubusercontent.com/31413407/52964467-5f7cc900-33c8-11e9-94e4-9129c61f5d8b.png)

Image -2

### CODE REVIEW

File: src/components/Component.vue [hunk 1]
```diff
@@ -246,10 +253,37 @@ A watcher has been added to the component to render the details dynamically when
 					this.sniper_data = result.data.sniper_data;
 					this.js_dependencies = result.data.js_dependencies;
 					this.css_dependencies = result.data.css_dependencies;
-					this.isLoading = false;
 					const url = `${API_URL}details/${this.name}`;
 					console.log(this.sniper_data);
-
+					axios({
+						method: 'GET',
+						url: this.githubURL.slice(0, 8) + 'api.' + this.githubURL.slice(8, 18) + '/repos' + this.githubURL.slice(18) + '/contents/README.md'
```
Comment (sarthak-sehgal) [inline src/components/Component.vue]: A better way to find the API url would be to split the Github url with '/' (let's say you split and store the value in an array named arr)
Then append arr[3] to arr[length-1] to "https://api.github.com/"

**Comment by sarthak-sehgal** [comment]
screenshot 2019-02-23 at 8 33 05 pm

The content is overflowing sometimes

**Comment by Nikhil-Vats** [comment]
@sarthak-sehgal I have fixed everything you suggested, please review and proceed.

**Comment by sarthak-sehgal** [comment]
The GitHub API you've used returns the raw README - it is plain text without syntax highlighting and markdown styling. Can we somehow have markdown styling?

**Comment by Nikhil-Vats** [comment]
@sarthak-sehgal I tried using markdown styling from github but we have to use their whole markdown css file for that which is not publically available. Also the API I have used **shows** README in markdown format which is slightly different from github's style. Maybe you saw plain text for a component because its README was actually just plain text, please try to see README of some other components to notice that they are in markdown style.

**Pull request merged**